smoker/src/smoker/smoker/commands.go

```diff
@@ -346,6 +378,116 @@ func genComponent(id uint) (backends.Component, error) {
 	}
 }
 
+// *** Auth Commands
+func replLogin(args []string, b backends.Backend) {
+	if len(args) == 0 {
+		fmt.Println("Please enter your username to login!")
+		return
+	} else if len(args) > 1 {
+		fmt.Println("You didn't enter your password in plaintext did you? You naughty boy.")
+		return
+	} else {
+		user := args[0]
+		password := GetPWs()
+		cred := backends.NewDummyCredential(user, password, backends.Unknown)
+		err := b.GetCredentialManager().Login(cred)
+		if err != nil {
+			color.Red("Error: " + err.Error())
+		} else {
+			color.Green("Login Successfull!")
+		}
+	}
+}
+
+func replAddUser(args []string, b backends.Backend) {
+	if !checkAdmin(b) {
```

Comment:
Checking if we have admin in the repl is a terrible idea. 

Need to move this into the dummy backend instead and throw a standard error.